### PR TITLE
Handle duplicate email error on signup

### PR DESCRIPTION
**Pull request by transhapHigsn**
Solution for #18

### CODE REVIEW

**Review by cfahlgren1 — CHANGES_REQUESTED** [review]
I pulled down your fork and I am getting the ```Something went wrong while registering you``` even when submitting a valid email. It also isn't showing up on the backend. Do you have any idea what might be the issue? Thank you for adding tests to prevent the issues in the future.

File: courts/views.py [hunk 1]
```diff
@@ -97,15 +97,24 @@ def simple_upload(request):
...
         if form.is_valid():
             new_signup = Signup()
             new_signup.email = email
-            new_signup.save()            
+            new_signup.save()
+        else:
+            # string matching like this is a bad solution.
+            # can't think of anything else, that's why going with this.
+            if 'Signup with this Email already exists.' in form.errors:
+                error_message = f'{email} is already registered.'
```
Comment (marcorichetta) [inline courts/views.py]: This assignment is always overwritten by the next line.

Comment (transhapHigsn) [reply courts/views.py]: Will change the order.

**Comment by transhapHigsn** [comment]
> I pulled down your fork and I am getting the `Something went wrong while registering you` even when submitting a valid email. It also isn't showing up on the backend. Do you have any idea what might be the issue? Thank you for adding tests to prevent the issues in the future.

I will check this and get back to you.

**Comment by transhapHigsn** [comment]
@marcorichetta sorry for the delay, i have fixed the code now.

**Review by cfahlgren1 — COMMENTED** [review]
When submitting correct email, it still presents an error. Maybe you can find where the error might be coming from? Make sure to pull down the new changes from the view.

**Comment by transhapHigsn** [comment]
> When submitting correct email, it still presents an error. Maybe you can find where the error might be coming from? Make sure to pull down the new changes from the view.

Okay, will check again.

**Pull request closed without merging**